README.md

```diff
@@ -0,0 +1,56 @@
+# Hello Microverse
+
+ Hello Microverse. This is our very first project at microverse. In this project we need to show that, we can use git and github smoothly. We would program a html page, which text color need to be changed using css.
```

Comment:
- [ ] Kindly, remove the description of the "Hello Microverse" project add a descriptive description of your project (Portfolio) in order to make your README file look more professional 😃. Your README is the entry point of your users and having it well written without any mistakes gives a good first impression and maximizes the potential of your product to your users.